### DISCUSSION TITLE
请教为啥登录还是显示Network Error？

### DISCUSSION THREAD

**Discussion by SkyPan19** [ZuodaoTech/everyone-can-use-english#852] [Q&A]
没有理解你说的“尝试科学上网”是啥意思，然后我去问了问AI，根据AI的提示，我重新检查了网络设置、清楚DNS缓存，但是还是登不进去？挺喜欢这个软件的，已经坚持一个多月了，昨天今天还是登不进去。

**Comment by an-lee** [comment]
因为域名 https://enjoy.bot 被污染了。正在想办法解决。

**Comment by an-lee** [comment]
解决办法： https://github.com/ZuodaoTech/everyone-can-use-english/discussions/853#discussioncomment-10101808
